### ISSUE TITLE
[Documentation]: Description of To check if the CNF have a reasonable startup time are not clear

### ISSUE THREAD

**Issue by CsatariGergely** [lfn-cnti/testsuite#1338]
**Describe the bug**
Test case description [To check if the CNF have a reasonable startup time](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md#heavy_check_mark-to-check-if-the-cnf-have-a-reasonable-startup-time) does not define what the reasonable time is and it does not define if it is valid for the whole CNF or for the specific Pod-s of the CNF. 

**Expected behavior**
1 The test case [description](https://github.com/cncf/cnf-testsuite/blob/main/USAGE.md#heavy_check_mark-to-check-if-the-cnf-have-a-reasonable-startup-time) should define what a reasonable startup time is. 
2 The test case description should define if this is valid for the whole CNF or for the specific services of the CNF. 
3 The rationale should explain the real issues of long startup time (e.g.: reliability) instead of pointing to "monolithic application"
4 The rationale should not mix CNF and the services of a CNF.

**Comment by kosstennbl** [comment]
Agree, the whole "reasonable_startup_time" still remains a mystery to me, it lacks clear description and good reasoning.
Possibly - this test should be reviewed and redesigned or removed altogether.

**Comment by Smitholi67** [comment]
Documentation has now been revised for clarity.

**Comment by Smitholi67** [comment]
Adding notes from CNTi Community Meeting on June 3, 2025.   For test result, take in account startup time of complete CNF. See “Helm install --wait” for criterias for helm-related CNFs and content of manifest in case of manifest-related CNFs. Increase threshold to a higher value (90s? - for further discussion, take in account results of execution in Sylva. Also consider various types of CNFs reasonable startup time (e.g. user plane, control plane, stateful, …)
Measure startup time of all k8s kinds inside the CNFs (pods, services, etc) and report as test results.
